File: src/main/java/tk/laspruca/tcplugin/util/DateGetter.java
package tk.laspruca.tcplugin.util;

import java.text.SimpleDateFormat;
import java.util.Date;

public class DateGetter {
	public static final SimpleDateFormat FORMAT = new SimpleDateFormat("E yyyy/MM/dd:HH:mm:ss");

	public static String getDate() {
		return FORMAT.format(new Date());
	}
}

File: src/main/kotlin/nz/laspruca/tcplugin/util/DateGetter.kt
package nz.laspruca.tcplugin.util

import java.text.SimpleDateFormat
import java.util.*

private val format = SimpleDateFormat("E yyyy/MM/dd:HH:mm:ss")

/**
 * Gets the current data and time using 'E yyyy/MM/dd:HH:mm:ss' format
 */
fun getDate(): String = format.format(Date())

